data/workflow/templates/metaT_inputs.tmpl

```diff
@@ -1,14 +1,7 @@
-    "main_workflow.<WORKFLOW>_proj": <PROJNAME>,
+    "main_workflow.<WORKFLOW>_project_id": <PROJNAME>,
     "main_workflow.<WORKFLOW>_input_file": <INPUT_FILE_SINGLE>,
     "main_workflow.<WORKFLOW>_input_fq1":<INPUT_FQ1>,
     "main_workflow.<WORKFLOW>_input_fq2":<INPUT_FQ2>,
     "main_workflow.<WORKFLOW>_input_interleaved":<INTERLEAVED>,
-    "main_workflow.<WORKFLOW>_git_url": "https://github.com/microbiomedata/mg_annotation/releases/tag/0.1",
-    "main_workflow.<WORKFLOW>_url_root": "https://data.microbiomedata.org/data/",
-    "main_workflow.<WORKFLOW>_outdir": <OUTDIR>,
-    "main_workflow.<WORKFLOW>_resource": "NMDC-EDGE",
-    "main_workflow.<WORKFLOW>_database": "/expanse/projects/nmdc/refdata/",
-    "main_workflow.<WORKFLOW>_activity_id": <PROJNAME>,
-    "main_workflow.<WORKFLOW>_informed_by": <PROJNAME>,
-    "main_workflow.<WORKFLOW>_threads": 64,
-    "main_workflow.<WORKFLOW>_metat_folder": "/expanse/projects/nmdc/edge_app/test/metaT/"
+    "main_workflow.<WORKFLOW>_strand_type":<STRAND_TYPE>
```

Comment:
Need UI for the strand_type? If not, use a default value or delete this line.

Reply:
Not sure, i can definitely put a default, but UI would be nice? For NMDC automation it'll be read from metadata

Reply:
Use a default for now. We can add UI when needed.